src/frontend/src/lib/i18n/en.json

```diff
@@ -537,7 +537,7 @@
 			"estimated_inter_network": "Estimated Inter-network Fee",
 			"estimated_eth": "Estimated Ethereum Fee",
 			"max_fee_eth": "Max fee <small>(likely in &lt; 30 seconds)</small>:",
-			"convert_fee": "Swap fee",
+			"convert_fee": "Convert fee",
```

Comment:
Maybe "Conversion fee"? Or does it sound better "Convert fee"?

Reply:
"Conversion fee" is better, thanks for the tip!